libraries/rush-lib/src/utilities/PathConstants.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See the @microsoft/rush package's LICENSE file for license information.
+
+import { PackageJsonLookup } from '@rushstack/node-core-library';
+
+/**
+ * The currently-executing rush-lib package's root folder path.
+ */
+export const rushLibFolderRootPath: string = PackageJsonLookup.instance.tryGetPackageFolderFor(__dirname)!;
```

Comment:
If we substituted a hardwired `path.resolve(__dirname, '../../../')` would that be significantly more efficient than `PackageJsonLookup` here?  I'm thinking of cases such as tab-completion that eval relatively little code.

Reply:
Yes, but I'm going to be raising a PR that includes shipping a webpacked version of `rush-lib` in addition to the `lib` folder version, so a hardcoded path won't work.